File: app/src/main/java/kr/zagros/shwan/moviemvvm/Entities/MovieResponse.java
package kr.zagros.shwan.moviemvvm.Entities;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MovieResponse {

    @SerializedName("data")
    @Expose
    private List<Movie> data = null;
    @SerializedName("metadata")
    @Expose
    private Metadata metadata;

    public List<Movie> getData() {
        return data;
    }

    public void setData(List<Movie> data) {
        this.data = data;
    }

    public Metadata getMetadata() {
        return metadata;
    }

    public void setMetadata(Metadata metadata) {
        this.metadata = metadata;
    }

}

File: app/src/main/java/kr/zagros/shwan/moviemvvm/Entities/MovieResponse.kt
package kr.zagros.shwan.moviemvvm.Entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MovieResponse {

    @SerializedName("data")
    @Expose
    var data: List<Movie>? = null
    @SerializedName("metadata")
    @Expose
    var metadata: Metadata? = null

}
